setup.py

```diff
@@ -5,29 +5,29 @@
 import re
 
 from setuptools import setup, find_packages
-here = os.path.abspath(os.path.dirname(__file__))
 
-try:  # for pip >= 10
-    from pip._internal.req import parse_requirements
-except ImportError:  # for pip <= 9.0.3
-    from pip.req import parse_requirements
-
-# parse_requirements() returns generator of pip.req.InstallRequirement objects
-install_reqs = parse_requirements('requirements.txt', session=False)
-
-# reqs is a list of requirement
-reqs = [str(ir.req) for ir in install_reqs]
+requirements = [
+    'requests',
+    'pendulum>=2.0.2'
```

Comment:
2.x even still current? this could be very old

Reply:
">=2.0.2" is actually what we want: We do not want to restrict the version (to an upper limit) but it has to be >= 2 because there they renamed "Pendulum" to "DateTime" (this is the main class).

Reply:
`~=2.0.2` does essentially the same. it just limits the versions when they go up. so 2.0.x is ok. 2.1.x is not. also 3.0 is not ok. we also could use `~=2.0` to limit it to 2.x releases